frontend/src/APIClients/CampsAPIClient.ts

```diff
@@ -104,7 +104,9 @@ const updateCampSessions = async (
   updatedCampSessions: Array<UpdateCampSessionsRequest>,
 ): Promise<Array<CampSessionResponse>> => {
   try {
-    const { data } = await baseAPIClient.patch(`/camp/${campId}/session`, {
```

Comment:
This is what I meant by "had to change the format", for some reason using client.patch(...) has some strange behaviour in terms of setting/sending headers (authorization in our case), but this other format worked perfectly. I'm still not entirely sure why though, thoughts? I tried to emulate whereever else we used a .patch for anything, but after trying it all, auth headers were still not sending.

Reply:
i don't see any issue with this!